src/include/ceph_fs.h

```diff
@@ -350,9 +350,10 @@ enum {
 	// internal op
 	CEPH_MDS_OP_FRAGMENTDIR= 0x01500,
 	CEPH_MDS_OP_EXPORTDIR  = 0x01501,
-	CEPH_MDS_OP_VALIDATE   = 0x01502,
-	CEPH_MDS_OP_FLUSH      = 0x01503,
-	CEPH_MDS_OP_ENQUEUE_SCRUB = 0x01504
+	CEPH_MDS_OP_FLUSH      = 0x01502,
+	CEPH_MDS_OP_ENQUEUE_SCRUB  = 0x01503,
+	CEPH_MDS_OP_REPAIR_FRAGSTATS = 0x01504,
+	CEPH_MDS_OP_REPAIR_INODESTATS = 0x01505
```

Comment:
I know the ones you're changing are only used locally; but these op IDs in aggregate are shared. It's not a big deal right now I suppose.